testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -2283,6 +2286,61 @@ public void testExtendedClientPolicyIntefacesForCiba() throws Exception {
         assertEquals("Exception thrown intentionally", response.getErrorDescription());
     }
 
+    @Test
+    @EnableFeature(value = Profile.Feature.CIBA, skipRestart = true)
+    @AuthServerContainerExclude({REMOTE, QUARKUS})
+    public void testExtendedClientPolicyIntefacesForBackchannelTokenRequest() throws Exception {
```

Comment:
IMO It will be ideal if ClientPoliciesTest does not have any dependency on the CIBA feature. Is it possible to either move this test to the CIBATest class or add new test class specific to CibaTest?

Reply:
@mposolda Yes, I will move these tests to `CIBATest`.

Reply:
@mposolda Yes, I will move these tests to `CIBATest`.